LICENCE.txt

```diff
@@ -0,0 +1,17 @@
+The MIT License (MIT)
+Copyright (c) 2016-2020 Ordanis Sanchez Suero
```

Comment:
This License  only require 1 date year, that is the  current one when is edited. I think you can delete the other one.